mantidproject/mantid
# Upgrade to Qt 5.15

**Note to gatekeepers. Please merge mantidproject/conda-recipes#84 when this is merged.**

**Description of work.**

Updates Qt on all platforms to 5.15. This is the last release of the 5.x series.



**To test:**

* This should be test on all platforms ideally. Packages have been built to facilitate testing:
  * macOS/Linux: https://builds.mantidproject.org/job/build_packages_from_branch/154/
  * Windows: https://builds.mantidproject.org/job/build_packages_from_branch/158/
* Build and check workbench starts.
* Have a play to see if anything obvious is broken. 

Fixes #34265 





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
